src/ert/scheduler/slurm_driver.py

```diff
@@ -29,7 +29,7 @@ class JobStatus(Enum):
     COMPLETED = auto()
     RUNNING = auto()
     FAILED = auto()
-    CANCELED = auto()
+    CANCELLED = auto()
```

Comment:
This is still referred as `CANCELED` in JobQueue slurm.

Reply:
Should be CANCELLED though, and scheduler will crash hard with this typo in place. Maybe the slurm driver has never been able to kill realizations?